### DISCUSSION TITLE
Intermission board.

### DISCUSSION THREAD

**Discussion by justinaramian** [riffnshred/nhl-led-scoreboard#265] [Dev and Ideas]
I have no idea if this would be possible, but I was thinking it might be kind of cool to have some kind of intermission board. Maybe showing stats (Shots, TOI, etc). Also maybe a countdown clock that starts once the game clock hits 0:00 so you have a rough idea of how much time is left in the intermission, it obviously wouldn't be 100% accurate, but close enough.

**Comment by riffnshred** [comment]
Some issues are already open about that but I like the different stats idea. As for the countdown clock it is currently impossible with the current version as the way the rendering works, it is really CPU intensive.  The one Idea I got is to have the Green indicator get smaller and smaller over time until the next period starts. Still needs to be tested tho.

**Reply by mij2** [reply]
I'm very new to github, but would be up for helping with something like this.

**Comment by mij2** [comment]
I had been thinking about this and might poke around and see what I can come up with. Was also looking at doing a goal board similar to the penalty board. I'll see what I can come up with.

Edit: Looks like there is already a goal board, but I haven't seen it pop up.
